### PR TITLE
Convert API response JSON to Python dict

### PR DESCRIPTION
**Pull request by lucioseki**
Some methods in this library are returning the API response JSON
string without converting it to a Python dict.
This is forcing the caller to selectively call `json.loads`
depending on the retrieved resource (e.g. `users` resource requires
`json.loads`, but `items` doesn't).

This patch makes all the API responses to be converted to a Python
dict, so the caller can consistently use the return value as a dict
regardless the requested resource.

### CODE REVIEW

**Comment by lucioseki** [comment]
This PR addresses the issue #25

**Review by chdastolfo — APPROVED** [review]
👍

**Review by rhymiz — CHANGES_REQUESTED** [review]
@lucioseki my proposal here is to change `response.text` to `response.json()` in the graphql client. That's the source of the issue.

Note: `requests` will raise an exception if you call `.json` on a response object that does not contain a 2xx or 4xx status code, that can either be handle explicitly by you, or you can call `response.raise_for_status()` before moving on to calling `response.json`.

File: monday/resources/base.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import json
-
+from monday import __version__
+from monday.exceptions import MondayError
```
Comment (rhymiz) [inline monday/resources/base.py:2]: did you mean to import these?

Comment (lucioseki) [reply monday/resources/base.py:2]: oops, I didn't

**Review by rhymiz — APPROVED** [review]
This looks good! Thanks for addressing all the comments!

**Pull request merged**